dracut.sh

```diff
@@ -2154,6 +2154,8 @@ if [[ $early_microcode == yes ]]; then
                 done
                 for i in $_fwdir/$_fw/$_src; do
                     [[ -e $i ]] || continue
+                    # skip README
```

Comment:
```suggestion
                    # skip README*
```

Reply:
That's why I used `str_starts` with the full path instead of `std_ends`, to handle the different possible compressions. Do you think a `*` is really necessary in the previous comment?

Reply:
The `*` for me document that we are using `str_starts` because it's needed and not by accident.
Someone rewriting this code without context might think they only need to skip `README`